source/shared/smart-contracts/guides/setup-tools.rst

```diff
@@ -32,7 +32,14 @@ It can be used for :ref:`compiling<compile-module>` and
    Add links for testing and schemas.
 
 Cargo Concordium is distributed as part of the :ref:`Concordium software<downloads>` package.
-The tool should be placed in your PATH.
+The tool should be placed in your PATH. This can be achieved by either:
+
```

Comment:
```suggestion

* Rename the downloaded file to ``cargo-concordium`` for MacOS/Linux, and ``cargo-concordium.exe`` for Windows.
```

The original issue from our support forum also mentions the need to rename the file. This also ensures, that the file correctly replaces already existing versions when placed in the same folder.